File: public/class01/ts1.py
# ts1.py

# This script should plot a time series of prices.

import pandas as pd
import datetime

my_df    = pd.read_csv('http://www.spy611.com/csv/allpredictions.csv')
sp0_pred = my_df['cdate'] > '2016'
sp0_df   = my_df[sp0_pred]
cdate_sr = sp0_df['cdate']
cp_sr    = sp0_df['cp']
  
# matplotlib likes dates:
cdate_l = [datetime.datetime.strptime(row, "%Y-%m-%d") for row in cdate_sr]

import matplotlib.pyplot as plt
# cdate_l holds x values, cp_sr: y values
plt.plot(cdate_l, cp_sr)
plt.show()
'bye'

File: public/class05/ts1.r
# ts1.r

# This script should plot a time series.

# Demo:
# R --no-save
# then:
# source('ts1.r')

# I need only three lines of syntax:

csv_l = read.csv('http://www.spy611.com/csv/allpredictions.csv')[ , c('cdate','cp')]
my_d  = as.Date(csv_l$cdate, "%Y-%m-%d")
plot(csv_l$cp ~ my_d)





